src/metabase/api/database.clj

```diff
@@ -1042,6 +1043,18 @@
                                       [:in :collection_id (api/check-404 (seq (db/select-ids Collection :name schema)))])])
          (map api.table/card->virtual-table))))
 
+(api/defendpoint GET "/:id/models"
+  "Returns a list of models for a database."
+  [id]
+  {id su/IntGreaterThanZero}
+  (api/read-check Database id)
+  (prn "GET /api/database/:id/models" id)
+  (let [cards (-> (db/select Card, :database_id id, :archived false, :dataset true, {:order-by [[:%lower.name :asc]]})
+                  (hydrate :creator :collection))
+        cards (filter mi/can-read? cards)
+        last-edit-info (:card (last-edit/fetch-last-edited-info {:card-ids (map :id cards)}))]
+    (vec (for [c cards] (m/assoc-some c :last-edit-info (get last-edit-info (:id c)))))))
```

Comment:
```suggestion
  (let [cards          (-> (db/select Card, :database_id id, :archived false, :dataset true, {:order-by [[:%lower.name :asc]]})
                           (hydrate :creator :collection))
        card-ids       (map :id (filter mi/can-read? cards))
        last-edit-info (:card (last-edit/fetch-last-edited-info {:card-ids card-ids}))]
    (vec (for [{:keys [card-id] :as card} cards] 
           (m/assoc-some card :last-edit-info (get last-edit-info card-id))))))
```

Is this easier to read?

Reply:
It's bikeshedding, but I personally find this too verbose. Consider this https://github.com/clojure/core.async/blob/master/src/main/clojure/clojure/core/async.clj#L1062

Reply:
It does seem to conflict with the rest of our codebase though, which aligns `let` bindings and has the `for` body on a new line. So on second thought, I'll accept your suggestion.